doc/getting-started/quick_start.ipynb

```diff
@@ -50,142 +70,192 @@
    "source": [
     "## Table of contents\n",
     "\n",
-    "* Get tutorial data via git\n",
-    "* General Introduction\n",
-    "* Data preparation\n",
-    "* Prepare input script\n",
-    "* Train a model\n",
-    "* Freeze a model\n",
-    "* Test a model\n",
-    "* Run MD with LAMMPS"
+    "<div align=\"left\" style=\"margin:rem;\"><img src=\"https://docs.deepmodeling.com/projects/deepmd/en/master/_static/logo.svg\" alt=\"deepmd-kit\" style=\"width:20rem;\"></div>\n",
+    "\n",
+    "* [Background](#background)\n",
```

Comment:
These links seem not working: https://deepmodeling--3567.org.readthedocs.build/projects/deepmd/en/3567/getting-started/quick_start.html#data-preparation